### PR TITLE
APOLLO -24186 Part 2-Re-added About schema section

### PR DESCRIPTION
**Pull request by roblucar**
Based on suggestions from Matt Mitchell , re-added about schema. Removed "get" references

### CODE REVIEW

**Comment by roblucar** [comment]
Hi @mwmitchell  can you review the latest changes, I could not update my original PR, so if this looks good, can you merge all the way to master? I dont have write access. Thanks

File: java-sdk/configuration.asciidoc [hunk 1]
```diff
@@ -53,3 +53,46 @@ Once a connector configuration has been defined, it can be associated with the `
 From that point, the framework takes care of providing the configuration instances to your connector.
 It also generates the schema, and sends it along to Fusion when it connects to Fusion.
 
+=== About com.lucidworks.schema
+The `com.lucidworks.schema` project (included in the connectors SDK) aims to simplify the process of creating JSON Schemas.
```
Comment (mwmitchell) [inline java-sdk/configuration.asciidoc:57]:
```suggestion
The `com.lucidworks.schema` library, which is exposed by the connectors SDK, aims to simplify the process of creating JSON Schemas.
```

File: java-sdk/configuration.asciidoc [hunk 2]
```diff
@@ -53,3 +53,46 @@ Once a connector configuration has been defined, it can be associated with the `
...
+It also provides utilities for building type-safe POJOs from schema definitions.
+
+The basic idea here is that you create an interface that extends `Model`, add property getters, and then add a few annotations.
+Once your interface has been built, you can generate an `ObjectType` instance, which is the object that contains all of the JSON schema metadata.
+By then combining that schema object with a `Map<String,Object>`, you can create instances of your interface to use as configuration objects.
```
Comment (mwmitchell) [inline java-sdk/configuration.asciidoc:62]:
```suggestion
```

File: java-sdk/configuration.asciidoc [hunk 3]
```diff
@@ -53,3 +53,46 @@ Once a connector configuration has been defined, it can be associated with the `
 From that point, the framework takes care of providing the configuration instances to your connector.
 It also generates the schema, and sends it along to Fusion when it connects to Fusion.
 
+=== About com.lucidworks.schema
+The `com.lucidworks.schema` project (included in the connectors SDK) aims to simplify the process of creating JSON Schemas.
+It also provides utilities for building type-safe POJOs from schema definitions.
+
+The basic idea here is that you create an interface that extends `Model`, add property getters, and then add a few annotations.
+Once your interface has been built, you can generate an `ObjectType` instance, which is the object that contains all of the JSON schema metadata.
```
Comment (mwmitchell) [inline java-sdk/configuration.asciidoc:61]:
```suggestion
```

File: java-sdk/configuration.asciidoc [hunk 4]
```diff
@@ -53,3 +53,46 @@ Once a connector configuration has been defined, it can be associated with the `
...
+Most method calls to these proxy instances result in method calls to the underlying `Map<String,Object>`.
+For example, if the interface defines a `id` method, then a runtime call results in a call to the `Map<String,Object>`: `data.get("id")`.
+
+A few special cases exist:
+
+* `toString` is proxied directly to the `Map<String,Object>`
+* `getClass` returns the class of the interface provided to `ModelGenerator#generate()`
+* `_data` returns the underlying `Map<String,Object>` object
+* If the method starts with `set`, the arguments are captured and sent to the underlying map via `put()`
+
```
Comment (mwmitchell) [inline java-sdk/configuration.asciidoc:74]:
```suggestion
```

File: java-sdk/configuration.asciidoc [hunk 5]
```diff
@@ -53,3 +53,46 @@ Once a connector configuration has been defined, it can be associated with the `
 From that point, the framework takes care of providing the configuration instances to your connector.
 It also generates the schema, and sends it along to Fusion when it connects to Fusion.
 
+=== About com.lucidworks.schema
+The `com.lucidworks.schema` project (included in the connectors SDK) aims to simplify the process of creating JSON Schemas.
+It also provides utilities for building type-safe POJOs from schema definitions.
+
+The basic idea here is that you create an interface that extends `Model`, add property getters, and then add a few annotations.
```
Comment (mwmitchell) [inline java-sdk/configuration.asciidoc:60]:
```suggestion
The main idea is that you create an interface that extends `Model`, add property getter methods, and a few annotations to describe property constraints and validation rules.

A configuration type is associated to a connector by calling `ConnectorPlugin.builder(MyConnectorConfig.class)`, you can see an example link:java-sdk/connectors/random-connector/src/main/java/com/lucidworks/connector/plugins/random/RandomContentPlugin.java#L27[here]. This is what makes it possible for the runtime framework to create and provide instances of the configuration, or datasource, when the connector runs.

The connector runtime also uses these `Model` based interfaces to generate plugin specific schemas. Schemas are used by Fusion in many different ways, but perhaps the most useful is the Fusion Admin UI. The HTML forms used for creating and updating Fusion datasources are dynamically generated from plugin schemas.
```

**Pull request merged**